openvpn/session/authentication_test.go

```diff
@@ -0,0 +1,55 @@
+package session
+
+import (
+	"github.com/mysterium/node/identity"
+	"github.com/mysterium/node/session"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+var returnSessionNotFound = func(sessionId session.SessionID) (session.Session, bool) {
```

Comment:
Why not a simple function?
```go
func returnSessionNotFound(sessionId session.SessionID) (session.Session, bool) {
```

Reply:
Refactored to configurable structure with single method.